pwa-devdocs/_drafts/talons/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Talons
+---
+
+<!-- TODO: Publish this topic under src/peregrine -->
+
+Most [React component][] contain two distinct sections: a section for logic and a presentational section.
+
+The logic section contains code for generating new values from props, maintaining local state, and lifecycle or side effect methods.
+This section can be considered the "brains" of the component.
+
+The presentational section of a React component contains code that define the component's DOM structure.
+It often uses values from the logic part of the component to pass down into another component or display using HTML elements.
+
+PWA Studio separates these two sections into distinct components with specific concerns: a Venia UI component and a Peregrine Talon.
+
+## Peregrine talon overview
+
+Peregrine talons are the logic component counterparts for Venia UI components.
+Each talon in Peregrine is designed to provide data or perform side effects for a specific Venia UI component.
+
+For example, the [`SearchBar`][] component uses the [`useSearchBar()`][] talon to get initial values and functions it needs to handle changes in focus or search term value.
+The `useSearchBar()` talon also keeps track of the component's `expanded` state, which determines how the search bar looks.
+
+## Talons versus hooks
+
+Talons are a PWA Studio concept that represent a specialized type of React hook.
+
+Peregrine **hooks**, such as `useQuery()` and `useWindowSize()`, are designed to be re-usable pieces of code, but
+Peregrine **talons** are designed to fulfill the logic needs of a single Venia UI component.
+This means that each talon belongs to a specific UI component in Venia and is not used in any other Venia component.
+
+For example, the functionality provided by the `useHeader()` talon is only relevant to the the `Header` component.
+
+## Using talons
+
+Talons are not re-usable hooks, but
+they make it easier to work with UI components by providing standard functionality.
+This lets developers focus on developing the presentation DOM and CSS pieces of a component.
+
+Venia UI components use talons by default, so
+developers can use Venia UI components as is without worrying about talons.
+If developers want to change how a Venia UI component looks, they can import its associated talon into a new or modified UI component of the same type.
```

Comment:
The extension/customization story isn't quite complete but maybe we want to show an example?

Reply:
I would rather wait until the story is complete before adding more information about it